app_legacy/Helpers/database/set-claim.php

```diff
@@ -232,13 +232,13 @@ function getClaimData(int $gameID, bool $getFullData = true): array
  * above stats and returning data for a specific user or game.
  */
 function getFilteredClaims(
-    int $gameID = 0,
+    int $gameID = null,
     int $claimFilter = ClaimFilters::AllFilters,
     int $sortType = ClaimSorting::ClaimDateDescending,
     bool $getExpiringOnly = false,
     ?string $username = null,
-    int $offset = 0,
-    int $limit = 50
+    int $offset = null,
+    int $limit = null
```

Comment:
Shouldn't these be `?int`?

Reply:
Good catch, thanks!
PHPStan in level 5 (current) does not catch those. I'd love to increase the level to at least 6/7 at some point which would highlight such inconsistencies, too.